infrastructure/ecs_task/task_role.tf

```diff
@@ -45,13 +20,3 @@ resource "aws_iam_role" "task_role" {
     workspace = "${terraform.workspace}"
   }
 }
-
-resource "aws_iam_policy" "task_policy" {
```

Comment:
Where are these added now? This is the ecs task policy, should be added here, no?